### PR TITLE
fix commonName using const

### PR DESCRIPTION
**Pull request by dsy3502**
using const for string “KubeEdge”

### CODE REVIEW

**Comment by gy95** [comment]
welcome, :) please signoff the commit to pass the DCO
ref: https://github.com/kubeedge/kubeedge/pull/3472/checks?check_run_id=4533842935

File: keadm/cmd/keadm/app/cmd/edge/join.go [hunk 1]
```diff
@@ -18,6 +18,7 @@ package cmd
 
 import (
 	"fmt"
+	"github.com/kubeedge/kubeedge/common/constants"
```
Comment (gy95) [inline keadm/cmd/keadm/app/cmd/edge/join.go:21]: kubeedge own package should be placed in the third part. For example: line 28 29

Comment (dsy3502) [reply keadm/cmd/keadm/app/cmd/edge/join.go:22]: I've modified him

Comment (dsy3502) [reply keadm/cmd/keadm/app/cmd/edge/join.go:22]: I've modified it

Comment (gy95) [reply keadm/cmd/keadm/app/cmd/edge/join.go:22]: also need fix the lint error, and please squash the commits to only one.

**Comment by zc2638** [comment]
/kind cleanup
And please fix the CI error

**Pull request closed without merging**